web/client/components/geostory/navigation/Navigation.jsx

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2019, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import Toolbar from '../../misc/toolbar/Toolbar';
+import {ButtonToolbar} from 'react-bootstrap';
+
+export default ({
```

Comment:
Hi, I've seen that documentation of this component is missing. may you consider to add it? Thanks